mne/io/edf/tests/test_edf.py

```diff
@@ -538,6 +538,17 @@ def test_read_latin1_annotations(tmp_path):
     annot_file = tmp_path / "annotations.txt"
     with open(annot_file, "wb") as f:
         f.write(annot)
+
+    # Test reading directly from file
+    onset, duration, description = _read_annotations_edf(
```

Comment:
I was going to ask you to use `mne.read_annotations` directly, but there is no `encoding` argument there. Given that it can matter and that we have one for `read_raw_edf`, it probably makes sense to add it here. Could you add it to `read_annotations` and use `read_annotations` directly here instead?